src/main/docker/backend/Dockerfile

```
@@ -1,9 +1,9 @@
-FROM cosmtrek/air:v1.65.1 AS dev
+FROM cosmtrek/air:v1.67.3 AS dev
 
 RUN go install github.com/go-delve/delve/cmd/dlv@latest
```

Hadolint (2.14.0)

[warning] Pin versions in go. Instead of `go install <package>` use `go install <package>@<version>`

(DL3062)